### PR TITLE
prototype

### PR DESCRIPTION
**Pull request by TheRickParker**

### CODE REVIEW

File: reference/standard-Jenkinsfile [hunk 1]
```diff
@@ -14,11 +14,18 @@ pipeline {
         def SFDX_DEV_HUB_USERNAME = "${env.SFDX_DEV_HUB_USERNAME}"
         def SFDX_DEV_HUB_HOST = "${env.SFDX_DEV_HUB_HOST}"
 
-        def JWT_KEY_CRED_ID = "${env.JWT_CRED_ID_DH}"
+        def SFDX_PKG_HUB_USERNAME = "${env.SFDX_PKG_HUB_USERNAME}"
+        def SFDX_PKG_HUB_HOST = "${env.SFDX_PKG_HUB_HOST}"
```
Comment (ImJohnMDaniel) [inline reference/standard-Jenkinsfile]: @TheRickParker -- No changes are required to this reference file.

File: src/SfdxProjectBuilder.groovy [hunk 2]
```diff
@@ -859,8 +885,8 @@ class SfdxProjectBuilder implements Serializable {
 
   private void authenticateToDevHub() {
     _.echo('Authenticate to the Dev Hub ')
-    // _.echo(_.env.JWT_CRED_ID_DH)
-    _.withCredentials( [ _.file( credentialsId: _.env.JWT_CRED_ID_DH, variable: 'jwt_key_file') ] ) {
+    // _.echo(_.env.JWT_KEY_CRED_ID_DH)
+    _.withCredentials( [ _.file( credentialsId: _.env.JWT_KEY_CRED_ID_DH, variable: 'jwt_key_file') ] ) {
```
Comment (ImJohnMDaniel) [inline src/SfdxProjectBuilder.groovy:873]: We will need to remember to change the environment variable on this in the Jenkins config

File: src/SfdxProjectBuilder.groovy [hunk 3]
```diff
@@ -90,6 +90,31 @@ class SfdxProjectBuilder implements Serializable {
   // the parsed contents of the SFDX project's configuration
   private def sfdxPackage
 
+  // dynamic property scaffold
+  private String packagingAlias() {
```
Comment (ImJohnMDaniel) [inline src/SfdxProjectBuilder.groovy]: @TheRickParker, I am not seeing this method used anywhere.  Am I missing something?

File: src/SfdxProjectBuilder.groovy [hunk 4]
```diff
@@ -90,6 +90,31 @@ class SfdxProjectBuilder implements Serializable {
...
+    if ( _.env.SFDX_PKG_HUB_USERNAME?.trim() && _.env.SFDX_PKG_HUB_HOST?.trim() ) {
+      return _.env.SFDX_PKG_HUB_ALIAS
+    } else {
+      return _.env.SFDX_DEV_HUB_ALIAS
+    }
+  }
+
+  private String packagingHost() {
```
Comment (ImJohnMDaniel) [inline src/SfdxProjectBuilder.groovy]: @TheRickParker, I am not seeing this method used anywhere.  Am I missing something?

**Review by ImJohnMDaniel — CHANGES_REQUESTED** [review]

File: reference/standard-Jenkinsfile (hunk 1 above)
Comment (TheRickParker) [reply reference/standard-Jenkinsfile]: I was under the impression that this was required in order to receive the variables. Should I revert these changes?

File: src/SfdxProjectBuilder.groovy (hunk 3 above)
Comment (TheRickParker) [reply src/SfdxProjectBuilder.groovy]: You aren't missing anything. I prepped them "just in case".

File: src/SfdxProjectBuilder.groovy (hunk 4 above)
Comment (TheRickParker) [reply src/SfdxProjectBuilder.groovy]: You aren't missing anything. I prepped them "just in case".

File: src/SfdxProjectBuilder.groovy (hunk 2 above)
Comment (TheRickParker) [reply src/SfdxProjectBuilder.groovy:873]: I was under the impression these were the variables defined in the reference file. I see now what you mean. Was twisted in my brain, I suppose.

**Pull request closed without merging**